samples/open-telemetry/customizing/Core_9/Sample/NetHostProcessor.cs

```diff
@@ -6,12 +6,7 @@
 
 class NetHostProcessor : BaseProcessor<Activity>
```

Comment:
@lailabougria is this worth keeping? It feels that we are showing an OTEL feature that has nothing to do with the NServiceBus? (The machine name is already visible in the traces via the `nservicebus.originating_machine` header)

Reply:
Discuessed with @lailabougria and we decided to remove